website/src/hugo/content/changelog.md

```diff
@@ -8,6 +8,119 @@
 Maintenance branches are merged before each new release. This change log is
 ordered chronologically, so each release contains all changes described below it.
 
+# v0.22.8 (2021-12-07)
+
+This is a maintenance release, binary compatible with the 0.22.x series.
+
+## http4s-core
+
+* Update case-insensitive, ... to 1.2.0 in series/0.22 by @scala-steward in https://github.com/http4s/http4s/pull/5434
+* Sec-Fetch-Dest header by @m3is0 in https://github.com/http4s/http4s/pull/5357
```

Comment:
Were there concerns about this PR, before it get branded in the binary record?
https://github.com/http4s/http4s/pull/5357#issuecomment-951899791

Reply:
Yeah, and it got reverted, and GitHub's automation didn't recognize that.  It also misses the cherry picks, which I guess we should turn into PRs if we're going to start doing things with this tool.